jsapp/js/components/activity/activityLogs.query.ts

```diff
@@ -139,10 +108,21 @@ const startActivityLogsExport = async () =>
  * @param itemLimit Pagination parameter: number of items per page
  * @param pageOffset Pagination parameter: offset of the page
  */
-export const useActivityLogsQuery = (itemLimit: number, pageOffset: number) =>
+export const useActivityLogsQuery = ({
+  limit,
+  offset,
+  projectId,
+  actionFilter,
+}: PaginatedQueryHookParams) =>
   useQuery({
-    queryKey: [QueryKeys.activityLogs, itemLimit, pageOffset],
-    queryFn: () => getActivityLogs(itemLimit, pageOffset),
+    queryKey: [QueryKeys.activityLogs, projectId, actionFilter, limit, offset],
+    queryFn: () =>
+      getActivityLogs({
+        projectId: projectId as string,
```

Comment:
Is `projectId` actually an `assetUid`? We always use `assetUid` name and never use `projectUid` or `projectId` or `assetId` (ok, the last one can be found in some ancient code).